gemini/agent-engine/tutorial_plangen_on_naturalplan_examples.ipynb

```diff
@@ -0,0 +1,2325 @@
are the existing schedules for everyone during the day: \\nDouglas has meetings on Monday during 12:00 to 12:30, 15:00 to 15:30; \\nLawrence has meetings on Monday during 10:30 to 12:00, 13:00 to 13:30, 14:00 to 14:30, 15:30 to 16:00; \\nIsabella is busy on Monday during 9:00 to 12:30, 13:30 to 17:00; \\n\\nFind a time that works for everyone's schedule and constraints. \\nSOLUTION: Here is the proposed time: Monday, 12:30 - 13:00 \\n\\nTASK: You need to schedule a meeting for Joshua, Denise and Jeremy for one hour between the work hours of 9:00 to 17:00 on Monday. \\n\\nHere are the existing schedules for everyone during the day: \\nJoshua is busy on Monday during 10:00 to 10:30, 12:00 to 12:30, 14:00 to 14:30, 15:00 to 15:30; \\nDenise's calendar is wide open the entire day.\\nJeremy has meetings on Monday during 9:30 to 10:30, 12:00 to 13:00, 13:30 to 14:00, 14:30 to 15:00, 15:30 to 16:00, 16:30 to 17:00; \\n\\nFind a time that works for everyone's schedule and constraints. \\nSOLUTION: Here is the proposed time: Monday, 10:30 - 11:30 \\n\\nTASK: You need to schedule a meeting for Alan, Elizabeth and Denise for half an hour between the work hours of 9:00 to 17:00 on Monday. \\n\\nHere are the existing schedules for everyone during the day: \\nAlan has blocked their calendar on Monday during 13:00 to 13:30, 14:00 to 14:30; \\nElizabeth is busy on Monday during 9:00 to 9:30, 11:00 to 11:30; \\nDenise has blocked their calendar on Monday during 9:00 to 10:00, 10:30 to 11:30, 12:30 to 13:00, 13:30 to 14:00, 14:30 to 16:00, 16:30 to 17:00; \\n\\nYou would like to schedule the meeting at their earlist availability.\\nFind a time that works for everyone's schedule and constraints. \\nSOLUTION: Here is the proposed time: Monday, 10:00 - 10:30 \\n\\nTASK: You need to schedule a meeting for Mason, Bruce and Christopher for half an hour between the work hours of 9:00 to 17:00 on Monday. \\n\\nHere are the existing schedules for everyone during the day: \\nMason has meetings on Monday during 9:30 to 10:00, 11:00 to 11:30, 14:30 to 15:00, 16:30 to 17:00; \\nBruce is free the entire day.\\nChristopher is busy on Monday during 9:30 to 10:30, 11:30 to 12:30, 15:00 to 17:00; \\n\\nMason would rather not meet on Monday before 12:30. Find a time that works for everyone's schedule and constraints. \\nSOLUTION: Here is the proposed time: Monday, 12:30 - 13:00 \\n\\nTASK: You need to schedule a meeting for Michelle, Steven and Jerry for one hour between the work hours of 9:00 to 17:00 on Monday. \\n\\nHere are the existing schedules for everyone during the day: \\nMichelle has meetings on Monday during 11:00 to 12:00; \\nSteven has blocked their calendar on Monday during 9:00 to 9:30, 11:30 to 12:00, 13:30 to 14:00, 15:30 to 16:00; \\nJerry has blocked their calendar on Monday during 9:00 to 9:30, 10:00 to 11:00, 11:30 to 12:30, 13:00 to 14:30, 15:30 to 16:00, 16:30 to 17:00; \\n\\nFind a time that works for everyone's schedule and constraints. \\nSOLUTION: \",\n",
```

Comment:
Can you load these large data prompts into separate files or use Google Cloud Storage? We have a GCS bucket for large files which can be imported.